lcd.cpp

```diff
@@ -485,14 +501,16 @@ void LCD_driveZ() {
 void LCD_driveF() {
   if (lcd_click_now) MENU_POP();
 
-  if (lcd_turn) {
-    // protect servo, don't drive beyond physical limits
-    float newF = feed_rate + lcd_turn > 0 ? 1 : -1;
-    if (newF < MIN_FEEDRATE) newF = MIN_FEEDRATE;
-    if (newF > MAX_FEEDRATE) newF = MAX_FEEDRATE;
-    // move
-    feed_rate = newF;
-  }
+  // if knob was spun quickly, then increment quicker
+  int multiplier = 1;
+  if ( lcd_turns_to_process > 2 ) multiplier = 10;
+
+  // protect servo, don't drive beyond physical limits
+  float newF = feed_rate + (lcd_turns_to_process * multiplier);
```

Comment:
when i turn positive, multiplier=10.  when i turn negative, multiplier=1.  this is intentional?

Reply:
nope, that needs to be fixed.  That should be an  abs compare so it does 10x for either direction.